### ISSUE TITLE
PyTorch out of CPU memory?

### ISSUE THREAD

**Issue by jeffcrouse** [openai/jukebox#86]
When running the first sampling command:
```
python jukebox/sample.py --model=5b_lyrics --name=sample_5b --levels=3 --sample_length_in_seconds=20 \
--total_sample_length_in_seconds=180 --sr=44100 --n_samples=6 --hop_fraction=0.5,0.5,0.125
```

I get this error

`RuntimeError: [enforce fail at CPUAllocator.cpp:56] posix_memalign(&data, gAlignment, nbytes) == 0. 12 vs 0`

According to [this issue](https://github.com/pytorch/pytorch/issues/31302), that means I am out of CPU memory. But I am using a P5000 Paperspace machine with 30GB of memory, which I assumed would be more than enough.  How much is required?

If this isn't sufficient, can anyone suggest a cloud provider that has a machine powerful enough to run jukebox?

Thanks!

**Comment by youdontown** [comment]
I would like some system requirements because I am running out of RAM as well (I have 24 GB).

**Comment by johndpope** [comment]
VRAM not RAM. 
Side note - 24 gb of VRAM ddr5 - eBay sell cards for $300 USD nvidia k80 / you’ll need a server with fans to run it. Or 24gb of ddr6 vram ~ nvidia rtx Titan 8000 ~ $3500 USD? 

New cards coming out soon nvidia 3080 ti

Instead /  you can use google colab / TPUs instead or rent a GPU on vast.ai / need to make it clearer for people in readme that this isn’t so hobby friendly for people wanting to turn on and start spitting out new music. If you have access to gpu resources / I’d be happy to help configure. I have some boutique sounds I want to do priming on.

**Comment by jeffcrouse** [comment]
Yes, FWIW, I signed up for Google Collab Pro and I'm having more luck than on a Paperspace server. Slower, but about 100x less expensive, so that's fine with me for now. For me personally, 

I originally found the Collab link in this issue https://github.com/openai/jukebox/issues/40  I haven't  explored if there are any differences yet, but maybe something about restoring sampling if your session gets interrupted?
